types.go

```diff
@@ -28,6 +28,11 @@ import (
 
 // Scope is a namespace wrapper around a stats reporter, ensuring that
 // all emitted values have a given prefix or set of tags.
+//
```

Comment:
this probably belongs in the separate prob-related PR as well since it's not related to the mutexes

Reply:
👍